assets/src/edit-story/components/inspector/karma/prepublishSelect.karma.js

```diff
@@ -141,13 +141,12 @@ describe('Pre-publish checklist select offending elements onClick', () => {
         MESSAGES.CRITICAL_METADATA.MISSING_POSTER.MAIN_TEXT
       );
 
-      expect(recommendedIssue).not.toBeNull();
-      expect(highPriorityIssue).not.toBeNull();
+      expect(recommendedIssue).toBeDefined();
+      expect(highPriorityIssue).toBeDefined();
```

Comment:
I don't know if this matters but usually I use `toBeInTheDocument`. Honestly - not sure if that matcher works in our karma tests, but this is good!